packages/dashboard-frontend/src/containers/Loader/Factory/Steps/Fetch/Devfile/buildStepTitle.ts

```diff
@@ -29,10 +29,9 @@ export default function (
     newTitle = `Devfile could not be found in ${sourceUrl}. Applying the default configuration.`;
   } else {
     newTitle = `Devfile found in repo ${sourceUrl} as '${source}'.`;
-  }
-  if (factoryResolverConverted.isConverted) {
-    const devfile = factoryResolverConverted.devfileV2;
```

Comment:
why can't we use `factoryResolverConverted.devfileV2` ?

Reply:
or could `factoryResolverConverted.devfileV2.schemaVersion` sometimes contain 2.2.0 (or else 2.1.0) ?